core/templates/dev/head/services/ExplorationFeaturesService.ts

```diff
@@ -24,17 +24,21 @@ import { downgradeInjectable } from '@angular/upgrade/static';
   providedIn: 'root'
 })
 export class ExplorationFeaturesService {
+  static isInitialized = false;
```

Comment:
Let's call this serviceIsInitialized to differentiate it from the similarly-named isInitialized() methods elsewhere.

Reply:
Done.